x509-cert/tests/examples/crates.io-chain.pem

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
You might also consider adding a preamble here, similar to the tests for `pem-rfc7468`:

```suggestion
This is an X.509 certificate chain serialized as PEM, with a preamble.

```